### PR TITLE
Add better example for make rule

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/make.rst [hunk 1]
```diff
@@ -74,14 +74,30 @@ Custom rules
 ------------
 
 In the ``<package>.mk`` file we can create some other rules.
-Here is a simple example that copies a file:
+Here is a simple example that create the `apache/mapcache.xml` file
```
Comment (asaunier) [inline doc/integrator/make.rst]: create => creates

File: doc/integrator/make.rst [hunk 2]
```diff
@@ -74,14 +74,30 @@ Custom rules
...
+and remove it on the `clean` rule:
```
Comment (asaunier) [inline doc/integrator/make.rst]: remove => removes

File: doc/integrator/make.rst [hunk 3]
```diff
@@ -74,14 +74,30 @@ Custom rules
...
 
-    $(DESTINATION_FILE): <source file>
-        cp $< $@
+    apache/mapcache.xml: tilegeneration/config.yaml .build/dev-requirements.timestamp
+        $(VENV_BIN)/generate_controller --mapcache
+
+    clean: project-clean
+    .PHONY: project-clean
+    project-clean:
+        rm -tapache/mapcache.xml
```
Comment (asaunier) [inline doc/integrator/make.rst]: missing blank space after "-t"
By the way is this "t" option correct? I don't see it in the "rm" command help page.

File: doc/integrator/make.rst [hunk 4]
```diff
@@ -74,14 +74,30 @@ Custom rules
...
+
+The ``tilegeneration/config.yaml`` and ``.build/dev-requirements.timestamp`` files are files that
```
Comment (asaunier) [inline doc/integrator/make.rst]: I would rather write:

``tilegeneration/config.yaml`` and ``.build/dev-requirements.timestamp`` are the files that must be built to run the current rule.

File: doc/integrator/make.rst [hunk 5]
```diff
@@ -74,14 +74,30 @@ Custom rules
...
+are required to build to do the current rule.
+
+The ``$(VENV_BIN)/generate_controller --mapcache`` is the command that generate the
+wanted file.
```
Comment (asaunier) [inline doc/integrator/make.rst]: ``$(VENV_BIN)/generate_controller --mapcache`` is the command that generateS the wanted file.

File: doc/integrator/make.rst [hunk 6]
```diff
@@ -74,14 +74,30 @@ Custom rules
...
+
+.. note::
+
+   The ``.build/*.timestamp`` files are not relay required files but they are flags
```
Comment (asaunier) [inline doc/integrator/make.rst]: relay => really

The ``.build/*.timestamp`` files are not really required  but they are flags indicating that an other rule is correctly done.

**Comment by asaunier** [comment]
Thanks @sbrunner for this really useful doc!
A few typos :P 
In the commit message as well: fot => for
Add better example for make rule

File: doc/integrator/make.rst (hunk 3 above)
Comment (sbrunner) [inline doc/integrator/make.rst]: It should -f :-)

**Comment by sbrunner** [comment]
Thanks @asaunier for your review :-)

**Comment by asaunier** [comment]
Looks good to me!

**Pull request merged**